src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
I just reverted my changes to main.css and the :target style look like this:
`:target {
    background: #FDFFD3;
    background-color: #FDFFD3;
}`

Unfortunately, it doesn't solve the issue. Is this the change that you had in mind?